src/main/java/com/novare/inventoryManager/menus/adminMenu/AdminMenuController.java

```diff
@@ -0,0 +1,90 @@
+package com.novare.inventoryManager.menus.adminMenu;
+
+import com.novare.inventoryManager.auth.Registrator;
+import com.novare.inventoryManager.employees.Employee;
+import com.novare.inventoryManager.employees.EmployeeRole;
+import com.novare.inventoryManager.utils.Menu;
+
+import java.math.BigDecimal;
+import java.security.NoSuchAlgorithmException;
+import java.util.Scanner;
+
+class AdminMenuController {
+    private final AdminMenuModel model;
+    private final AdminMenuView view;
+    private final Scanner scanner;
+    AdminMenuController(AdminMenuModel model, AdminMenuView view) {
+        this.model = model;
+        this.view = view;
+        this.scanner = new Scanner(System.in);
+    }
+
+    void requestUserInput() {
+        String input = scanner.nextLine();
+
+        try {
+            int selectedOption = Integer.parseInt(input);
+            handleOption(selectedOption);
+        }
+        catch (NumberFormatException | IndexOutOfBoundsException exception) {
+                Menu.printInvalidOption();
+        }
+        catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    void handleOption(int selectedOption) throws IndexOutOfBoundsException, NoSuchAlgorithmException {
+        try {
+            switch (selectedOption) {
+                case 1 -> addNewEmployee(EmployeeRole.MANAGER);
+                case 2 -> addNewEmployee(EmployeeRole.CASHIER);
+
```

Comment:
Admin should be able to see the list of employee based on the UseCase diagram

Reply:
to do